docs/Cloning/index.rst

```diff
@@ -28,3 +28,6 @@ are optional.
 
 .. note::
     Note: Only unauthenticated clones are supported over https. If you do not specify the CLONE_PROTOCOL, or specify it as ssh, you must provide the associated ssh configurations and keys.
+
+Here is an example command using some environment variables::
+    docker run --env AUTODETECT=true --env VCS_DOMAIN=github.com --env CLONE_REPOSITORIES=terraform-aws-modules/terraform-aws-security-group --env CLONE_PROTOCOL=https seiso/easy_infra:latest-terraform
```

Comment:
I would trim out anything that's not completely necessary for the example.  Also, (a nit) to shorten the example and make it more readable I recommend `-e` instead of `--env`.  What do you think?

```suggestion
    docker run -e VCS_DOMAIN=github.com -e CLONE_REPOSITORIES=terraform-aws-modules/terraform-aws-security-group -e CLONE_PROTOCOL=https seiso/easy_infra:latest-terraform
```

Reply:
If you agree with adding `scan_terraform` we should probably describe in the sentence beforehand that this will clone and then scan a repo.

Reply:
Thanks for the feedback. I'll incorporate the changes. I'll provide a more readable example with just the essential environment variables. I'll also include an action that results in a scan with an explanation of what it's doing. Thanks again.